.github/workflows/deploy-pypi.yml

```
@@ -0,0 +1,18 @@
+name: Publish to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+jobs:
+  build-and-test-sdist-and-wheels:
+    uses: glotzerlab/signac/.github/workflows/publish-packages.yml@feature/actions_deploy_pypi
```

Reminder to fix this before merging.
```suggestion
    uses: glotzerlab/signac/.github/workflows/publish-packages.yml
```